### PR TITLE
[15.0][MIG] agreement_rebate: Migration to version 15.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT36438

ping @carlosdauden @CarlosRoca13

### CODE REVIEW

File: agreement_rebate/migrations/13.0.1.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+]
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.rename_tables(env.cr, tables_to_rename)
+    openupgrade.remove_tables_fks(env.cr, [tables_to_rename[0][1]])
+    openupgrade.rename_columns(
+        env.cr, {"agreement_rebate_settlement": [("invoice_id", None)]}
+    )
```
Comment (carlosdauden) [inline agreement_rebate/migrations/13.0.1.0.0/pre-migration.py]: Remove this file

File: agreement_rebate/migrations/13.0.1.0.0/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                FROM account_invoice ai
+                WHERE ai.id = ars.{}"""
+        ).format(sql.Identifier(openupgrade.get_legacy_name("invoice_id"))),
+    )
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    update_sale_order_line_invoice_rel(env.cr)
+    update_agreement_rebate_settlement_invoice(env.cr)
```
Comment (carlosdauden) [inline agreement_rebate/migrations/13.0.1.0.0/post-migration.py]: Remove this file

**Review by carlosdauden — CHANGES_REQUESTED** [review]
Remove migration unneeded folder

**Comment by sergio-teruel** [comment]
@carlosdauden Changes done!!

**Review by CarlosRoca13 — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**